pyiron_contrib/atomistics/atomistics/job/trainingcontainer.py

```diff
@@ -231,7 +231,7 @@ def plot(self):
         """
         :class:`.TrainingPlots`: plotting interface
         """
-        return TrainingPlots(self)
+        return TrainingPlots(self._container)
```

Comment:
This is nice!

Reply:
Should I merge #317 into this one?

Reply:
I will merge it to have everything in one pr